package.json

```diff
@@ -4,7 +4,8 @@
   "description": "JSON Web Token implementation (symmetric and asymmetric)",
   "main": "index.js",
   "scripts": {
-    "test": "nyc --reporter=html --reporter=text mocha && nsp check && cost-of-modules"
+    "test:unit": "nyc --reporter=html --reporter=text mocha",
```

Comment:
🤔 they are not unit tests. I guess you wanted a fast way to develop the tests getting coverage feedback without the delay of nsp + cost-of-modules, right? since this is test + coverage, let's name it: `coverage` or `test:coverage` or similar.

Reply:
Sure thing! `cost-of-modules` temporarily renames the `node_modules/` directory, and it was causing my editor to continually re-index. This was a way to run the tests without those checks.

I'm okay with a `coverage` task.